src/meta/src/barrier/command.rs

```diff
@@ -257,6 +374,15 @@ where
                     )
                     .await?;
             }
+
+            Command::RescheduleFragment(reschedules) => {
+                // TODO: drop actors on worker nodes.
```

Comment:
If `dropped_actors` in `UpdateMutation` is not empty, the corresponding actor will be dropped after this barrier flow

Reply:
Exactly, the tokio task for these actors will exit automatically. However, we used to drop some other records about this actor in a separated RPC of `drop_actor`. I believe we can find a way to avoid this redundant step?

https://github.com/singularity-data/risingwave/blob/aea0007bb07cc2681eeddc796987bd2321b9d13f/src/stream/src/task/stream_manager.rs#L618-L628